docs/internet-identity-spec.adoc

```diff
@@ -650,9 +650,8 @@ Similar to the steps during initial installation. The main difference is that yo
 
 ==== On Mainnet
 
-Write a proposal description like those in
+Write a proposal description like those in <https://github.com/dfinity/nns-proposals/blob/main/proposals/network_canister_management/>. Use a recent Internet Identity upgrade proposal as a template. It contains a consensed `git log` of the changes since the last release.
```

Comment:
```suggestion
Write a proposal description like those in <https://github.com/dfinity/nns-proposals/blob/main/proposals/network_canister_management/>. Use a recent Internet Identity upgrade proposal as a template. It contains a condensed `git log` of the changes since the last release.
```